students/mielczarek_szymon/lesson_05_lists/swap_min_and_max.py

```diff
@@ -0,0 +1,12 @@
+a = [int(i) for i in input().split()]
+min_idx = 0
+max_idx = 0
+
+for i in range(1, len(a)):
```

Comment:
good, 
however, in my opinion, it would be better and more readable to avoid iteration and use following:
`min_idx = index(min(a))`
so the full solution would have less lines:
```
a = [int(i) for i in input().split()]

min_idx = a.index(min(a))
max_idx = a.index(max(a))

a[min_idx], a[max_idx] = a[max_idx], a[min_idx]
print(' '.join([str(i) for i in a]))
```